tests/Feature/InitializeGitRepositoryTest.php

```diff
@@ -2,20 +2,21 @@
 
 namespace Tests\Feature;
 
-use App\Actions\InitializeGitRepo;
+use App\Actions\InitializeGitRepository;
 use App\LamboException;
 use Tests\Feature\Fakes\FakeProcess;
 use Tests\TestCase;
 
-class InitializeGitRepoTest extends TestCase
+class InitializeGitRepositoryTest extends TestCase
 {
     /** @test */
-    function it_initialises_the_projects_git_repository()
+    function it_initialises_git_with_the_specified_branch_name()
```

Comment:
Shouldn't we keep the original test that asserts it works when not customizing the initial branch?

Reply:
You're right. I had implemented `config('lambo.store.branch')` to be configured in `App\Commands\NewCommand` with _main_ as its value by default and overridden if `--branch` is provided. I had removed the original test since I knew it would always have a value. 

However, your comment has me reconsidering. I think I want it to be null by default.

Reply:
Done